package.json

```diff
@@ -1,25 +1,24 @@
 {
-  "name": "sample-template-node",
+  "name": "sample-conversations-masked-numbers",
   "version": "1.0.0",
   "private": true,
   "main": "./app.js",
   "scripts": {
     "start": "node .",
     "setup": "configure-env",
-    "pretest": "cross-env NODE_ENV=test",
-    "test": "eslint . && mocha test",
+    "test": "eslint . && cross-env NODE_ENV=test mocha test",
     "format": "eslint . --fix",
     "dev": "nodemon ."
   },
-  "description": "Node.js template for Twilio sample apps",
+  "description": "2-way Masked Phone Number SMS Conversation",
   "author": {
     "name": "Twilio",
     "email": "open-source@twilio.com"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/twilio-labs/sample-template-node"
+    "url": "https://github.com/TwilioDevEd/sample-conversations-masked-numbers"
```

Comment:
This should be `twilio-labs` instead of `TwilioDevEd` as we'll move it there

Reply:
👍